fr.openfisca.org/legislation-explorer/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+cd /home/openfisca/legislation-explorer
+git pull
+npm install
+
+# Ideally, we would like to take the .env into account, but it doesn't work yet. See https://github.com/openfisca/legislation-explorer/pull/171
+NODE_ENV=production PATHNAME=/legislation/  API_URL=https://fr.openfisca.org/api/v22 npm run build
```

Comment:
`npm run build` should be enough once https://github.com/openfisca/legislation-explorer/pull/173 (or equivalent) is merged